### PR TITLE
feedback from Greg

### PR DESCRIPTION
**Pull request by peterb-teamwork**
# This is a Teamwork passion project and not officially supported by the team

The creators and maintainers of this project are doing it with enthusiasm and happiness, in odd moments between other projects and possibly even in their spare time. Please be kind to them, and understand that although they hope to get the time & energy to fix your bug :crossed_fingers:, this is not an officially supported product or service, and therefore doing so is not a part of their day jobs.

## Checklist

- [] changes include new automated tests covering them
- [] have tested manually yourself
- [] PR describes why you made the change, not just how
- [] PR describes where where we should look to best understand changes & their effects (in code, in the resulting running software, elsewhere?)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,12 +1,3 @@
----
-name: Bug report
-about: Create a report to help us improve
-title: ''
-labels: ''
-assignees: ''
-
----
-
```
Comment (peterb-teamwork) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]: There are fields in the template and Github features for these things.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -1,18 +1,9 @@
----
-name: Feature request
-about: Suggest an idea for this project
-title: ''
-labels: ''
-assignees: ''
-
----
-
```
Comment (peterb-teamwork) [inline .github/ISSUE_TEMPLATE/feature_request.md:9]: There are fields in the template and Github features for these things.

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -1,18 +1,9 @@
...
 **This is a Teamwork passion project and not officially supported by the team**
 
 The creators and maintainers of this project are doing it with enthusiasm and happiness, in odd moments between other projects and possibly even in their spare time. Please be kind to them, and understand that although they hope to get the time & energy to understand & implement your suggestion :crossed_fingers:, this is not an officially supported product or service, and therefore doing so is not a part of their day jobs.
 
 **Name it**
-If this request contains multiple features then list them as bullet points.
+Please give it a meaningful name, and please put separate requests in separate issues. If this request contains multiple closely-related features that should be considered together then list them as bullet points.
```
Comment (peterb-teamwork) [inline .github/ISSUE_TEMPLATE/feature_request.md:6]: Encourage smaller units of change that are more likely to work well.

**Review by aaaaaaaalex — DISMISSED** [review]
Looking good :)

**Review by aaaaaaaalex — DISMISSED** [review]

**Review by aaaaaaaalex — APPROVED** [review]

**Pull request merged**